packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.base.tsx

```diff
@@ -490,7 +490,8 @@ export class DropdownBase extends React.Component<IDropdownInternalProps, IDropd
           direction={FocusZoneDirection.vertical}
           id={id + '-list'}
           className={this._classNames.dropdownItems}
-          aria-labelledby={label ? id + '-label' : undefined}
+          aria-label={ariaLabel}
+          ariaLabelledBy={label && !ariaLabel ? id + '-label' : undefined}
```

Comment:
I think you should also fix FocusZone to respect either `aria-labelledby` or `ariaLabelledBy`--one way is to move the settings up a few lines ([code link](https://github.com/OfficeDev/office-ui-fabric-react/blob/master/packages/office-ui-fabric-react/src/components/FocusZone/FocusZone.tsx#L231-L233)):
```
      <Tag
        role="presentation"
        aria-labelledby={ariaLabelledBy}
        aria-describedby={ariaDescribedBy}
        {...divProps}
```

Would be interesting to see if this problem also occurs in other components.